microgen/phase.py

```diff
@@ -125,26 +124,29 @@ def solids(self) -> list[cq.Solid]:
             print("No solids or shape")
             return []
```

Comment:
??? This `return []` should never happen i think

Reply:
Yes, this merge request does not change **any** behaviour except for the bug fix